packages/playwright-test/src/reporter/SerenityReporterForPlaywrightTest.ts

```diff
@@ -40,26 +60,25 @@ import { SERENITY_JS_DOMAIN_EVENTS_ATTACHMENT_CONTENT_TYPE } from './PlaywrightA
  * See {@apilink SerenityOptions} for usage examples.
  */
 export interface SerenityReporterForPlaywrightTestConfig {
-
     /**
-     * A list of {@apilink StageCrewMemberBuilder|StageCrewMemberBuilders} or {@apilink StageCrewMember|StageCrewMembers}
-     * to be instantiated in Playwright Test reporter process and notified of {@apilink DomainEvent|DomainEvents} that occur during the scenario execution.
-     * Note that the `crew` can also be configured using {@apilink ClassDescription|ClassDescriptions}.
-     *
-     * #### Learn more
-     * - {@apilink SerenityOptions}
-     * - {@apilink SerenityConfig.crew}
-     */
+   * A list of {@apilink StageCrewMemberBuilder|StageCrewMemberBuilders} or {@apilink StageCrewMember|StageCrewMembers}
```

Comment:
Looks like formatting has gone a bit off; maybe some editor settings that kicked in as part of the commit?

Reply:
seems like it, maybe I run auto formatter. Fixed docs are pushed